boards/opentitan/src/tests/sip_hash.rs

```diff
@@ -63,40 +62,152 @@ impl<'a> hasher::Client<'a, 8> for SipHashTestCallback {
         assert_eq!(result, Ok(()));
         // Value calculated from:
         //    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=08a45842acb9dc1abf9dbc303b5eaa50
-        assert_eq!(ret, 0x9ed5975598371f51);
+        if self.run_count.get() == 0 {
+            assert_eq!(ret, 0x9ed5975598371f51);
+        } else if self.run_count.get() == 1 {
+            assert_eq!(ret, 0xB5326A7D96F8A5B7);
+        } else {
+            panic!();
```

Comment:
Probably worth adding a message here

Reply:
Do you mean like a debug message output to say that the assertion was ok for each of the runs?

Reply:
`panic!("Unsupported number of callbacks");`